ui/job-view/details/BugFiler.jsx

```diff
@@ -583,7 +583,7 @@ export class BugFilerClass extends React.Component {
                   ))}
                 </FormGroup>
               </div>
-              <label>Summary:</label>
+              <Label htmlFor="summary">Summary:</Label>
```

Comment:
It looks like Reactstrap allows you to use the `for` attribute in the Label component.